src/ActivityStickyQueues/Program.cs

```diff
@@ -0,0 +1,74 @@
+using Microsoft.Extensions.Logging;
+using Temporalio.Client;
+using Temporalio.Worker;
+using TemporalioSamples.ActivityStickyQueues;
+
+// Create a client to localhost on default namespace
+var client = await TemporalClient.ConnectAsync(new("localhost:7233")
+{
+    LoggerFactory = LoggerFactory.Create(builder =>
+        builder.
+            AddSimpleConsole(options => options.TimestampFormat = "[HH:mm:ss] ").
+            SetMinimumLevel(LogLevel.Information)),
+});
+
+async Task RunWorkerAsync()
+{
+    // Cancellation token cancelled on ctrl+c
+    using var tokenSource = new CancellationTokenSource();
+    Console.CancelKeyPress += (_, eventArgs) =>
+    {
+        tokenSource.Cancel();
+        eventArgs.Cancel = true;
+    };
+
+    var uniqueWorkerTaskQueue = Guid.NewGuid().ToString();
+
+    var nonStickyActivities = new NonStickyActivities(uniqueWorkerTaskQueue);
+    var stickyActivities = new StickyActivities();
+
+    // Run worker until cancelled
+    Console.WriteLine("Running worker");
+
+    using var nonStickyWorker = new TemporalWorker(
+        client,
+        new TemporalWorkerOptions(taskQueue: "sticky-activity-tutorial")
```

Comment:
```suggestion
        new TemporalWorkerOptions(taskQueue: "activity-sticky-queues-sample")
```

Matches our other examples better